_nubia/nubianx569j.markdown

```diff
@@ -0,0 +1,38 @@
+---
+layout: device
+title:  "Nubia ZTE Z17 mini"
+codename: nx569j
+downloadfolder: nx569j
+supportstatus: Current
```

Comment:
I'm leaving it as current because it seems like new versions of TWRP are released for this device (the latest version [here](https://eu.dl.twrp.me/nx569j/) is from 2021-11-28)